content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -38,15 +38,16 @@ See the Google Kubernetes Engine (GKE) guide to [configuring cluster access for
 
     ```
     export PIPELINE_VERSION={{% pipelines/latest-version %}}
-    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
+    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/base/crds?ref=$PIPELINE_VERSION
     kubectl wait --for condition=established --timeout=60s crd/applications.app.k8s.io
-    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
     ```
     **Note**: The above approach is introduced in Kubeflow Pipelines version 0.2.0. For older versions please run the following instead:
     ```
     export PIPELINE_VERSION=<older_version>
-    kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
+    kubectl apply -k github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
     ```
+    **Note**: the URLs after `apply -k` are not real. They are [hashicorp/go-getter URL format supported by kustomize](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#url-format).
```

Comment:
I think it might be confusing to tell users that these URLs aren't "real". How about something like this:

```
**Note**: URLs after `apply -k` are in [hashicorp/go-getter URL format](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#url-format). 
```

Reply:
I meant, if they paste the urls into a browser to verify if the url is working, they get github 404 errors.
How can we convey this message?

Reply:
How about something like:

**Note**: `kubectl apply -k` accepts paths that are formatted as [hashicorp/go-getter URLs](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#url-format). While the paths in the preceding commands look like URLs, the paths are not valid URLs.

Reply:
Thanks! That's awesome. I will udpate

Reply:
Updated, WDUT?